app/helpers/application_helper.rb

```diff
@@ -183,4 +183,11 @@ def get_facility_locations_by_visn(visn)
     end
     location_list
   end
+
+  def ga_private?
+    params["controller"] == "admin" || # all admin pages
+    (params["controller"] == "page" && !@page&.published?) || # unpublished PageBuilder pages
+    (params["controller"] == "practices" && !["show", "index"].include?(params["action"])) || # practice editor pages
+    (params["controller"] == "practices" && params["action"] == "show" && !@practice.is_public) # internal or unpublished practices
```

Comment:
I'm wondering if we should add the user pages, like `recommended_for_you`, `users/show`, etc.